workers/src/main/java/edu/cornell/repository/JUnit5RepositoryImpl.java

```diff
@@ -19,7 +21,15 @@ final class JUnit5RepositoryImpl extends Repository {
         super(rootDir);
     }
 
+    /**
+     * Run JUnit Test Runner
+     * @param tests the list of names of test classes to run
+     * @param output the output stream to log test results to
+     */
     @Override
-    public void test(@NonNull List<String> tests, @NonNull TestOutputStream output) {
+    public void test(@NonNull List<String> tests, TestOutputStream output) {
```

Comment:
Discussed offline but we might want to hook up the `TestOutputStream` as well in this PR